### DISCUSSION TITLE
[Question]: A chatgpt error

### DISCUSSION THREAD

**Discussion by HeLongaa** [danny-avila/LibreChat#1663] [Troubleshooting]
### What is your question?

When using chatgpt, I entered a correct token,
but the program prompted "Something went wrong. Here's the specific error message we accounted for: Error message:" Failed to send message. HTTP 429- {"detail": "It is recommended to upgrade to the latest PandoraNext:" https://github.com/pandora-next/deploy "
My program is deployed using Huggingface. How should I handle this error?

### More Details

A chatgpt error

### What is the main subject of your question?

Other

### Screenshots

![image](https://github.com/danny-avila/LibreChat/assets/71657914/3c666caf-2efe-4638-b46b-8559b01ba084)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
See the pandoranext guide: https://docs.librechat.ai/features/pandoranext.html

**Reply by danny-avila** [reply]
Note this is for backend access to chat.openai.com and is not recommended as your messages may be seen/recorded by a 3rd party, it's not an officially supported service, and you may be banned by OpenAI for doing so.

If you are simply trying to access GPT (OpenAI) models, and confusing the ChatGPT option as such, you should follow the setup guide for using their official API: https://docs.librechat.ai/install/configuration/ai_setup.html#openai

**Reply by zimmra** [reply]
@danny-avila It is no longer possible to get PandoraNext license IDs, the maintainer has shut down the service as of today. I assume this means using ChatGPT tokens is no longer possible at this time?

- https://github.com/pandora-next/deploy/issues/519
- https://linux.do/t/topic/1051
- https://linux.do/t/topic/6181

**Reply by fuegovic** [reply]
@zimmra Thanks for the information, time to say goobye to the chatgpt (browser) endpoint 🪦

**Reply by danny-avila** [reply]
Translated message by the maintainer (google translate):

> Years ago, OpenAI released their epoch-making product: ChatGPT, and the whole world went crazy for it.
> 
> But in their eyes, China (and of course some countries) seems to be outside the scope of this "world". They block Chinese IP access, block Chinese mobile phone number registration, block Chinese bank card payments, and even block countless computer room IP and VPN manufacturers, and the number is increasing every day.
> 
> Maybe they have a reason why they have to do it, but I don't think it should be like this.
> 
> About a year ago (2023/02/17 21:58), I made the first commit to the Pandora project, and since then I have started to bypass various blocks of OpenAI. In April 2023, I built a ChatGPT sharing site for netizens who were unable to register an account. It serves netizens who want to experience the charm of ChatGPT for free, without advertising and without diverting traffic. Perhaps because of this, it has received the attention and support of many enthusiastic netizens, quickly expanded thousands of accounts, and the number of daily visits has reached one million or even tens of millions.
> 
> In this year, I have competed with OpenAI countless times: passing CF, passing 5-second shield, passing arkose, logging in, registering, etc. I wrote various sub-services and even trained dozens of CAPTCHA recognition models. In the constant struggle, the time, software and hardware costs I need to invest are constantly increasing (over 100 dollars/month). I am used to watching the service monitoring before going to bed every day. When I wake up, I still watch the service monitoring and report problems through private messages in group chats.
> 
> I have been well aware of this kind of reverse service since the beginning. If OpenAI changes a parameter, I will lose my leg, but I still enjoy it. Because with a little investment from me, thousands of Chinese people (and of course Russian brothers) can experience the convenience and efficiency improvements brought by advanced technology. I began to slowly understand Mr. Lu Xun’s words: “Infinite distances and countless people are all related to me.”
> 
> Of course this won't last. Recently, I realized that OpenAI has been targeting me and started to adjust and target frequently. Their adjustment frequency has begun to be measured in "days" or even "hours". The time and money I need to maintain this service have also increased dramatically, which makes me feel insignificant and powerless, and I can no longer afford it. So I understand that I have already reached the end of this matter.
> 
> Some friends said that I should charge an appropriate fee to maintain this project. Yes, this project has no charges, no commercial cooperation, and no advertising from the beginning to the present. This doesn’t mean that I can’t charge, it doesn’t mean that no one wants me for business cooperation, and it doesn’t mean that there are no advertisements to hang on. As for charging, I said in the group that the majority of netizens are caught between GFW and OpenAI, and the cost is already high. I don’t want to take the opportunity to make another cut; for business cooperation, my personal capabilities and resources are limited, and it will be difficult for me to make money after serving the business. I devote my energy to serving netizens; as for advertising, I personally don’t like it.
> 
> I am a technical person with no realm and sourness. The above points are my original intentions. If I have to violate them, then I would rather not do them. So let the farewell come! OpenAI, you won, but I leveraged you at a cost far lower than yours. At least so far, you haven’t killed Pandora, and I haven’t lost either.
> 
> The server bill will be due on January 30, 2024, and the server will be shut down at that time. Affected services include:
> 
>      chat*.zhile.io
>      chat-shared*.zhile.io
>      chat*.geekgpt.org
>      chat.oaifree.com 9.0k
>      *.pandoranext.com
>      ai*.fakeopen.com
> 
> I can only express my sincere apologies for the inconvenience caused to everyone (including ecological partners) by the shutdown of services and servers! Thank you all for your continued support. I hope that the pleasant moments in the process will become good memories, and I hope that you will not take the unpleasant ones to heart. All netizens who have donated free ChatGPT accounts, please also take back your accounts when the time comes. Thank you for your sharing spirit! Please bear with me if there is anything left unfinished.
> 
> Finally, I would like to remind you: Although Pandora makes you trust it, it does not mean that products with similar interfaces are worthy of trust. Please pay attention to screening in the future.
> 
> As for the future, I should do less continuous reverse engineering projects (which are really very time-consuming). From now on, I will spend more time with my family, chat and joke with netizens here (this thing is very good, you can post articles, comments, private messages, and real-time chat). Yes, he is such an ordinary person.
> 
> This moment has finally come 553
